### PR TITLE
chore: add decorator usage example to README

### PR DESCRIPTION
**Pull request by Birdi7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,15 @@ scalene_profiler.start()
 scalene_profiler.stop()
 ```
 
+To use Scalene as a decorator, import `scalene_redirect_profile`
+```Python
+from scalene.scalene_profiler import scalene_redirect_profile
```
Comment (emeryberger) [inline README.md:52]: Can just replace all this with `@profile` (no need to import `scalene_redirect_profile`, etc.).

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (emeryberger) [reply README.md:52]: Bump @Birdi7 - do you want to make a new PR or close this one?

Comment (Birdi7) [reply README.md:52]: I'll edit this PR, and bump you. Plan to do this during the next 5 days. Thank you for the bump :)

Comment (Birdi7) [reply README.md]: @emeryberger I've edited the PR to describe `@profile` usage

**Pull request merged**